INSTALL.md

```diff
@@ -1,75 +1,72 @@
 # Install -- overview
 
-Python 3.9 is required currently, to maintain compatibility with Google Colaboratory.
+Elf-simulations has been tested with Python 3.9 and 3.10.
 
-Set up your favorite python virtual environment. We use:
+1. Install Pyenv
 
-- [pyenv](https://github.com/pyenv/pyenv#how-it-works) to manage python versions
-- [venv](https://docs.python.org/3/library/venv.html) standard library to manage virtual environments
+Follow [Pyenv install instructions](https://github.com/pyenv/pyenv#installation).
 
-We also use [Docker](docs.docker.com/get-docker) for building images.
+2. Clone Elf-simulations repo
 
-# Install -- steps
+Clone the repo into a <repo_location> of your choice, i.e. `git clone https://github.com/delvtech/elf-simulations.git <repo_location>`
 
-Clone the repo into a <repo_location> of your choice.
-Next, follow the installation instructions provided by [pyenv](https://github.com/pyenv/pyenv#installation).
-
-After installation, we can use pyenv to install Python from within the repo.
+3. Set up virtual environment.
```

Comment:
please do not delete the links to pyenv and venv docs

Reply:
docker should also be in here. You could put it at the end in a note that it is required for deploying bot images or something

Reply:
added docker at the bottom. links to pyenv and venv are in their respective steps. i wanted to remove the preamble and jump into the steps so it's more obvious to the user what they need to do, without them skimming over anything.